module/mybooks.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+import sqlite3
+from telegram import Update
+from telegram.ext import CallbackContext
+from module.create_connection import create_connection
+from module.shared import DB_PATH, error_message
+from module.send_results import send_results
+
+
+def get_user_books(context: CallbackContext, chat_id: int) -> List:
```

Comment:
we could define the kind of `List`, if it is a `List[str]` or `List[int]` etc.

Reply:
or list of object like `List['className']`

Reply:
Fixed